bosh-setup/azuredeploy.json

```diff
@@ -66,6 +66,7 @@
   },
   "variables": {
     "apiVersion": "2015-06-15",
```

Comment:
remove these and hard-code the apiVersion in the resources - otherwise json editors cannot provide intellisense.